docs/src/design.md

```diff
@@ -35,7 +35,23 @@ Whenever it makes sense, four variants of the same operator are defined:
 
 Replace `something` with `derivative`, `multiderivative`, `gradient`, `jacobian`, `pushforward` or `pullback` to get the correct name.
```

Comment:
Let's emphasize with with an admonition (I personally read over it).

Reply:
And maybe change "primal too" to "differential and primal", or at least swap the rows.

Reply:
sounds good

Reply:
EDIT: "primal and differential"